File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int N = sc.nextInt();
        double[] arr = new double[N];
        
        for(int i = 0; i < N; i++) {
            arr[i] = sc.nextDouble();
        }
        
        Arrays.sort(arr);
        double sum = 0;
        
        for(int i = 0; i < N; i++) {
            sum += arr[i] / arr[N - 1]* 100;
        }
        
        System.out.println(sum / N);
    }
}

File: 백준/Bronze/1546. 평균/평균.cs
using System;

class Q1546
{
    static void Main(String[] args)
    {
        int N = int.Parse(Console.ReadLine());
        double[] arr = new double[N];
        
        String[] S = Console.ReadLine().Split();
        
        for(int i = 0; i < N; i++)
        {
            arr[i] = double.Parse(S[i]);
        }
        
        Array.Sort(arr);
        double sum = 0;
        
        for(int i = 0; i < N; i++)
        {
            sum += arr[i] / arr[N - 1] * 100;
        }
        
        Console.WriteLine(sum / N);
        
    }
}